packages/uiweb/src/lib/components/supportChat/ChatInput.tsx

```diff
@@ -40,22 +39,28 @@ export const ChatInput: React.FC = () => {
     e.preventDefault();
     setLoading(true);
     if (message.trim() !== '' && connectedUser) {
-      const sendResponse = await PushAPI.chat.send({
-        messageContent: message,
-        messageType: 'Text',
-        receiverAddress: supportAddress,
-        account: account,
-        pgpPrivateKey: connectedUser?.privateKey,
-        apiKey,
-        env,
+      
+      const sendResponse = await userAlice.chat.send(supportAddress ,{
+        
+        type: 'Text',
+        content: message,
+      
       });
+      console.log(sendResponse);
+      if (!sendResponse) {
+        setToastMessage(sendResponse);
+        setToastType('error');
+        setLoading(false);
+        
+      }
 
       if (typeof sendResponse !== 'string') {
-        sendResponse.messageContent = message;
-        setChatsSorted([...chats, sendResponse]);
+        // sendResponse.messageContent = message;
+        // setChatsSorted([...chats, sendResponse]);
```

Comment:
Remove commented code

Reply:
I am keeping this incase I want to shift to sorting message like this in the future. But rest every other comments I have removed